File: foundation/foundation 2 month/lesson1/homework/ex_5.c
#include <stdio.h>
int value(int num)
{
	if(num>=0)
	{
	}
	else
	num=num * -1;
	return num;
}
	int main()
{
	int n;
	printf("Sonni kiriting: ");
	scanf("%d",&n);
	printf("%d",value(n));
	return 0;
}

File: foundation/foundation 3 month/lesson9/homework/ex_5.py
import random
k=random.randint(1,100)
cnt=5
while cnt>0:
	s=int(input("Sonni kiriting: "))
	if s==k:
		print(f"Tugri topdigiz; {k},{cnt} urinishda")
		break
	else:
		cnt-=1
		print(f"{cnt} ta urinish qoldi ")
	if k>s:
		print("Siz kiritilgan sondan katta")
	elif k<s:
		print("Siz kiritgan sonndan kichik")
print(k)
